src/193/test/kotlin/org/rust/ide/hints/RsChainMethodHintsProviderTest.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.hints
+
+import com.intellij.codeInsight.hints.presentation.PresentationRenderer
+
+class RsChainMethodHintsProviderTest : RsChainMethodHintsProviderTestBase() {
+    override fun checkInlays() {
+        myFixture.testInlays(
+            { (it.renderer as PresentationRenderer).presentation.toString() },
+            { it.renderer is PresentationRenderer }
+        )
+    }
```

Comment:
Let's make some refactoring in hint tests to avoid code duplication
For example, you can create the following hierarchy:
`RsInlayTypeHintsTestPlatformBase` with single protected method `checkInlays`. Of course, you need two versions of it because of slightly different API in 2019.3 and 2020.
`RsInlayTypeHintsTestBase : RsInlayTypeHintsTestPlatformBase` with protected method `checkByText`. Right now we have two identical versions of it in `RsInlayTypeHintsProviderTestBase` and `RsChainMethodHintsProviderTestBase `
`RsInlayTypeHintsProviderTest : RsInlayTypeHintsTestBase` with all tests related to type hints (right now the corresponding class named `RsInlayTypeHintsTestBase`)
`RsChainMethodHintsProviderTest : RsInlayTypeHintsTestBase` with tests related to method chain hints
If you have ideas how to make it simpler and suggestions about better names - welcome